lib/syskit/cli/log_runtime_archive.rb

```diff
@@ -147,6 +160,23 @@ def find_last_archive_index(basename)
                 end
             end
 
+            def self.create_tcp_socket(host, port)
+                socket =
+                    begin TCPSocket.new(host, port)
+                    rescue Errno::ECONNREFUSED => e
+                        raise e.class, "cannot contact process server at " \
+                                       "'#{host}:#{port}': #{e.message}"
+                    end
+                socket.write(ProcessManagers::Remote::COMMAND_LOG_UPLOAD_FILE)
+                socket
```

Comment:
@doudou correct me if I am wrong, but she will need to use a Net::FTP client, right? I had told her to base herself on what the process server is doing, but this seems wrong now that I looking at it.

Reply:
@jhonasiv what exactly seems off here?

Reply:
I answered her in Chat, but for information's sake. This was used by process server to talk to each other, which is obviously not the case anymore.